app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
Interesting. I may have something wrong with the commenting. I'll check it tomorrow morning.